core/include/msg.h

```diff
@@ -363,6 +374,14 @@ int msg_reply_int(msg_t *m, msg_t *reply);
  */
 int msg_avail(void);
 
+/**
```

Comment:
can we rename this to sth like "msg_queue_capacity()"? at first sight, "queue length" is ambiguous with the current length of the queue, vs. its maximum length.